PowerPointLabs/Test/UnitTest/SyncLab/SyncLabPositionTest.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.Office.Interop.PowerPoint;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using PowerPointLabs.SyncLab.ObjectFormats;
+
+namespace Test.UnitTest.SyncLab
+{
+    [TestClass]
+    public class SyncLabPositionTest : BaseSyncLabTest
```

Comment:
Have u tried this https://github.com/PowerPointLabs/PowerPointLabs/blob/master/PowerPointLabs/Test/Util/MessageBoxUtil.cs